presto-thrift-connector/src/test/java/com/facebook/presto/connector/thrift/util/TestRetryDriver.java

```diff
@@ -75,28 +76,39 @@ void testNoRetry()
             throws Exception
     {
         SyncTaskImpl task = new SyncTaskImpl(SUCCESS);
-        Integer result = retry.run("test", task);
+        RetryStats retryStats = new RetryStats();
+        Integer result = retry.run("test", retryStats, task);
 
         assertSyncSuccess(result, task, 1);
+        assertEquals(retryStats.getSuccess().getTotalCount(), 1);
```

Comment:
This block is exactly the same in all tests. Think of adding a helper method. For example, you can add two: one for success, one for failure.
Then you'll do something like `assertSuccessfulStats(1); // number of attempts` and `assertFailedStats(3)`. Then move 4 lines you have into each of them.